src/pipecat/tests/utils.py

```diff
@@ -22,7 +22,8 @@
 from pipecat.pipeline.task import PipelineParams, PipelineTask
 from pipecat.processors.frame_processor import FrameDirection, FrameProcessor
 
-logger.remove(0)
+if logger._core.handlers:
+    logger.remove(0)
```

Comment:
What if we just do `logger.remove()`? Accessing private members is usually not a good idea.

Reply:
Another option is to simply remove these lines. I added them so I didn't have to do it in every test, but maybe it's better for people to control logging themselves.

Reply:
Yes, I think we want this last option. I'll create a PR for that.